frame.go

```diff
@@ -211,6 +217,17 @@ func (fr *frameImpl) Proto() *pb.Frame {
 	return fr.msg
 }
 
+func (fr *frameImpl) IsNull(index int, colName string) bool {
+	rowBitmask := fr.msg.Bitmask[index]
+	for _, nullColumnName := range rowBitmask.NullColumns {
```

Comment:
If we're going to check for membership and not care about order, shouldn't `NullColumns` be a set?